frontend/src/component/admin/maintenance/index.tsx

```diff
@@ -6,10 +6,11 @@ import { Box, styled } from '@mui/material';
 import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
 import { MaintenanceTooltip } from './MaintenanceTooltip';
 import { MaintenanceToggle } from './MaintenanceToggle';
+import { UPDATE_MAINTENANCE_MODE } from '@server/types/permissions';
```

Comment:
I didn't know we could import stuff from the server? Should it go in `AccessProvider/permissions` to be consistent with how we're doing it today? I think only defining it in one place is cool, though, but why is it not something we're already doing?

Reply:
Thanks for noticing this. It needs refactoring. I'll post another PR, because it turns out there are inconsistencies all over this.